Makefile

```diff
@@ -4,29 +4,29 @@ DOCKER_FILE=Dockerfile
 IMAGE_NAME=simulator_image
 
 docker-build:
-	docker build \
+	sudo docker build \
```

Comment:
Generally speaking, users / developers are recommended to use docker as non-root:
 - https://docs.docker.com/engine/install/linux-postinstall/
 - https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user